mod-orders/examples/bindPiecesResult.sample

```diff
@@ -0,0 +1,8 @@
+{
+  "poLineId": "3f4f61ab-d3c2-47a9-baa6-c903a4cfba57",
+  "boundPieceIds": [
+    "fe729e0b-5c56-4f0a-bb3a-33bc405d71a0",
+    "4d6b8cf9-73bd-4b3d-8f1b-5b7cbf02d03a"
+  ],
+  "requestsAction": "Transfer"
```

Comment:
I think request action is not useful in response, probably itemId would be more useful

Reply:
Okay then, will include it